### ISSUE TITLE
Bump package version & push latest --color support to NPM

### ISSUE THREAD

**Issue by obibring** [necinc/msvgc#9]
Hey there, the latest version on npm doesn't include support for `--color`. Can you update the version and publish?

**Comment by henrikra** [comment]
Yeah looks like the `--color` param has been merged to master but new release hasn't been done. I guess new release hasn't been done because of https://github.com/necinc/msvgc/issues/7 When I tried the latest version from master I got this error too :(
